### PR TITLE
FOGL-8509: changes to make all side menu options visible

### PR DESCRIPTION
**Pull request by MonikaSharma06**

### CODE REVIEW

**Review by praveen-garg — CHANGES_REQUESTED** [review]

File: src/app/components/layout/side-menu/side-menu.component.css [hunk 1]
```diff
@@ -1,11 +1,11 @@
 .menu-list {
   background-color: white !important;
-  min-height: 100vh;
-  margin-top: 0.25%;
+  width: 100%;
```
Comment (praveen-garg) [inline src/app/components/layout/side-menu/side-menu.component.css:3]: Why is this required?

Comment (MonikaSharma06) [reply src/app/components/layout/side-menu/side-menu.component.css:3]: this makes slightly change in width. But not very much required. I'll remove it.

**Review by praveen-garg — APPROVED** [review]

**Review by hemantgarg923 — APPROVED** [review]
